01-Git_Bash_Make/bash_script

```diff
@@ -1,7 +1,3 @@
 #!/bin/bash
 
 mkdir guess_a_number
-cp main.c guess_a_number/
-tar -czvf guess_a_number.tar.gz guess_a_number/
-mkdir guess_a_number/release
-cp guess_a_number.tar.gz guess_a_number/release/
```

Comment:
@vometel,
Currently commit message doesn't match its content.

I assume that the intent was to create empty script by the first commit of this PR
and fill it later by chain of commits.

Then please edit ef025cd57a7e (`Lesson 2 Part 2: Add bash script file`)
instead of removing these line here.